pr-data.csv

```diff
@@ -1259,6 +1259,11 @@ https://github.com/odrotbohm/sos,d4ed68a143b637c7e2d523edad1b105bf6c2d996,20-mod
 https://github.com/OpenAPITools/openapi-generator,b70edd7f1b600b7927913528405de98ebbebf5ce,modules/openapi-generator,org.openapitools.codegen.DefaultCodegenTest.verifyXDiscriminatorValue,ID,Opened,https://github.com/OpenAPITools/openapi-generator/pull/7958,
 https://github.com/OpenAPITools/openapi-generator,b70edd7f1b600b7927913528405de98ebbebf5ce,modules/openapi-generator,org.openapitools.codegen.kotlin.AbstractKotlinCodegenTest.convertModelName,ID,Opened,https://github.com/OpenAPITools/openapi-generator/pull/7958,
 https://github.com/OpenAPITools/openapi-generator,b70edd7f1b600b7927913528405de98ebbebf5ce,modules/openapi-generator,org.openapitools.codegen.kotlin.AbstractKotlinCodegenTest.convertVarName,ID,Opened,https://github.com/OpenAPITools/openapi-generator/pull/7958,
+https://github.com/OpenAPITools/openapi-generator,bfce822c34144500eb573ab105502c3616d79dc3,modules/openapi-generator,org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsSemiColon,ID,Accepted,https://github.com/OpenAPITools/openapi-generator/pull/10485,
+https://github.com/OpenAPITools/openapi-generator,bfce822c34144500eb573ab105502c3616d79dc3,modules/openapi-generator,org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsNewLineLinux,ID,Accepted,https://github.com/OpenAPITools/openapi-generator/pull/10485,
+https://github.com/OpenAPITools/openapi-generator,bfce822c34144500eb573ab105502c3616d79dc3,modules/openapi-generator,org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsNewLineWindows,ID,Accepted,https://github.com/OpenAPITools/openapi-generator/pull/10485,
+https://github.com/OpenAPITools/openapi-generator,bfce822c34144500eb573ab105502c3616d79dc3,modules/openapi-generator,org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsMixed,ID,Accepted,https://github.com/OpenAPITools/openapi-generator/pull/10485,
+https://github.com/OpenAPITools/openapi-generator,bfce822c34144500eb573ab105502c3616d79dc3,modules/openapi-generator,org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsNoDuplicate,ID,Accepted,https://github.com/OpenAPITools/openapi-generator/pull/10485,
```

Comment:
It is great that the developer agreed to merge the changes (but has yet to do so).

Can you please ensure that the build passes with your changes?
https://github.com/TestingResearchIllinois/idoft/pull/81/checks?check_run_id=3728028641

(E.g., one problem seems to be `org.openapitools.codegen.java.AbstractJavaCodegenTest,testAdditionalModelTypeAnnotationsNoDuplicate` should be `org.openapitools.codegen.java.AbstractJavaCodegenTest.testAdditionalModelTypeAnnotationsNoDuplicate`)

Reply:
Thank you pointing it out! Will look into this now.

Reply:
Interesting.. After changing the format, the `format-checker` still failed. I double-checked for several times but cannot figure out why my pr still violate the format.. 😅

Reply:
Thanks for your changes.

> ERROR: On file pr-data.csv: The file is not properly ordered
https://github.com/TestingResearchIllinois/idoft/pull/81/checks?check_run_id=3728150389#step:6:8

You can sort the file with the command in the following section of the readme: https://github.com/TestingResearchIllinois/idoft#to-contribute-a-newly-detected-flaky-test

Reply:
Sorry for not getting back to you sooner. Just sort the .csv file based on the document you shared and it works! This PR looks good now : )